### PR TITLE
Add Actix icon

### LINKED ISSUE #8453: Request: Actix Rust framework
### Brand Name

Actix

### Website

https://actix.rs/

### Popularity Metric

Category Rank -> 13,854 https://www.similarweb.com/website/actix.rs/#overview
Global Rank -> #794,526 https://www.similarweb.com/website/actix.rs/#overview

### Official Resources for Icon and Color

https://actix.rs/img/logo.png || color=#000000 or color=#ffffff || Taken from the header of the website

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by IamRezaMousavi**
actix

**Issue:** closes #8453 

 [18.4k GitHub ⭐](https://github.com/actix/actix-web)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by adamrusted** [comment]
@IamRezaMousavi is there an SVG version of the icon available on the website? All I can see are PNGs...

**Comment by IamRezaMousavi** [comment]
> @IamRezaMousavi is there an SVG version of the icon available on the website? All I can see are PNGs...

No, I converted the site icon from png to svg format

**Comment by IamRezaMousavi** [comment]
@adamrusted What do I need to do?

**Comment by IamRezaMousavi** [comment]
You can see https://crates.io/crates/actix-web 
This package has +120k downloads in past 4 days (4 Dec - 7 Dec)

**Comment by PeterShaggyNoble** [comment]
That looks like a good indicator to me, @IamRezaMousavi and puts it on a par with the currently proposed minimum downloads for npm. Would any of the other @simple-icons/maintainers agree that this is popular based on that data? Or should we discuss crates.io downloads further as part of our proposed new metrics first?

**Comment by service-paradis** [comment]
Looks pretty good to me!

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
On to reviewing, so 😁 Nice work on this one, @IamRezaMousavi; the differences are _mostly_ minor, which is to be expected with a manual vectorisation, but could you see if you could get the upper part of yours (in red) to line up a little better with the original?

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for updating, @IamRezaMousavi, but those differences look [more pronounced](https://github.com/simple-icons/simple-icons/assets/15157491/e4d2db09-ec0f-4d06-93d3-6dac17563bd3) to me now, unfortunately. Of the 2 versions you've tried so far, I think the first is probably our better option if that's as close as you can get 👍🏻

**Comment by PeterShaggyNoble** [comment]
In the interests of moving this along, I've reverted the SVG to the previous version. But I'd like one of the other @simple-icons/maintainers to have a look over it before proceeding.

File: icons/actix.svg
Comment (PratyushJayachandran) [inline icons/actix.svg:1]: There is a minor scaling issue. See:  
actix-comparison

Black being the PNG cropped end to end and red one being the contribution. The vector is pretty spot on but I guess the source was scaled with some extra pixels leading to overall change.

I am okay with it though. Would wait for other maintainers to comment.

Comment (adamrusted) [reply icons/actix.svg:1]: No, unfortunately I think the issues are too severe, even with lining up the PNG (in black) as closely as I could with the  SVG in red. Would you mind having another look please @IamRezaMousavi?

**Comment by service-paradis** [comment]
[Well, I found something that might be interesting](https://github.com/actix/actix-website/issues/18) 😅

**Comment by adamrusted** [comment]
> [Well, I found something that might be interesting](https://github.com/actix/actix-website/issues/18) 😅

**Review by service-paradis — APPROVED** [review]
LGTM now 🎉 

Thanks everyone that works on it!

**Pull request merged**